src/app/popular/layout.js

```diff
@@ -0,0 +1,7 @@
+import Page from "./page";
+
+export const metadata = {
+    title: "Popular Repositories - RepoRater",
```

Comment:
The indentation is not matching other changes and also the project settings, please check you have Prettier plugin install